template/private-npm-package/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "<%- fullModuleName %>",
+  "description": "<%- description %>",
+  "version": "0.0.0-semantically-released",
+  "main": "lib-commonjs",
+  "module": "lib-es2015",
+  "typings": "lib-types",
+  "files": [
+    "lib"
+  ],
+  "sideEffects": false,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "skuba build",
+    "commit": "skuba commit",
+    "format": "skuba format",
+    "format:check": "skuba format check",
```

Comment:
We can remove this as we've rolled `format:check` into `lint`, per https://github.com/seek-oss/skuba/blob/master/docs/migrating-from-seek-module-toolkit.md